service/build.gradle

```diff
@@ -32,61 +41,75 @@ dependencies {
     implementation group: "com.google.guava", name:"guava"
 
     // Get stairway via TCL
-    implementation "bio.terra:terra-common-lib" // 0.0.94-SNAPSHOT is defined in bio.terra.pipelines.java-common-conventions.gradle
+    implementation "bio.terra:terra-common-lib" // 0.1.9-SNAPSHOT is defined in java-common-conventions
+
+    // terra clients
+    implementation "org.broadinstitute.dsde.workbench:sam-client_2.13:0.1-5a5bf28"
+    implementation "org.broadinstitute.dsde.workbench:leonardo-client_2.13:1.3.6-22ee00b"
+    implementation "org.databiosphere:workspacedataservice-client-okhttp-jakarta:0.2.115-20240227.195850-1"
+    implementation "bio.terra:cbas-client:0.0.199"
     implementation "bio.terra:terra-resource-buffer-client:0.4.3-SNAPSHOT"
```

Comment:
just noticing this is in here, what is this and does it not need to be updated?

Reply:
nice catch, we don't use this - I think I added it by mistake when adding Stairway a while ago. will remove